src/general/g.citation/g.citation.py

```diff
@@ -639,8 +668,9 @@ def print_bibtex(citation):
 
     author_names = [author["name"] for author in citation["authors"]]
     print("  author = {", " and ".join(author_names), "},", sep="")
+    print("  howpublished = {", citation["code-url"], "},", sep="")
     print("  year = {", citation["year"], "}", sep="")
-
+    print("  note = {Accessed: ", citation["access"], "},", sep="")
```

Comment:
biblatex `@online` has `urldate`, maybe worth checking, although I'm not sure how applicable that is, but if you are adding accessed, it seems like it could fall under an online resource.

Reply:
I see, but could not find any specific with a quick search. Do you have any suggestion how or where to add that?

Reply:
Maybe leave that for a different PR then. There is anyway couple different systems and types which would make sense to include (misc, online, software, ...).